<launch>
    <arg name="gps_topic" default="/dji_sdk/gps_position" />
    <arg name="rtk_topic" default="/dji_sdk/rtk_position" />
    <arg name="attitude_topic" default="/dji_sdk/attitude" />
    <arg name="root_directory" default="/media/vant3d" />

    <node pkg="riser_inspection" type="mission_riser_node" name="riser_inspection_wp" output="screen">
        <param name="gps_topic" type="string" value="$(arg gps_topic)" />
        <param name="rtk_topic" type="string" value="$(arg rtk_topic)" />
        <param name="attitude_topic" type="string" value="$(arg attitude_topic)" />
        <param name="riser_distance" type="int" value="5"/>  <!--METERS-->
        <param name="riser_diameter" type="int" value="300"/>   <!--MILLIMETER-->
        <param name="horizontal_points" type="int" value="5"/>  <!--Nº of horizontal stops-->
        <param name="vertical_points" type="int" value="4"/>    <!--Nº of vertical stops at each horizontal point-->
        <param name="delta_H" type="int" value="15"/>           <!--DEGREES-->
        <param name="delta_V" type="int" value="300"/>          <!--MILLIMETERS-->
    </node>
</launch>